services/spar/Makefile

```diff
@@ -66,18 +66,18 @@ $(DEB_SCHEMA): install
 
 .PHONY: i
 i:
-	../integration.sh $(EXE_IT) -s $(NAME).integration.yaml -i ../integration.yaml $(WIRE_INTEGRATION_TEST_OPTIONS)
+	../integration.sh $(EXE_IT) -s $(NAME).integration.yaml -i ../integration.yaml -b ../brig/brig.integration.yaml  $(WIRE_INTEGRATION_TEST_OPTIONS)
 
 .PHONY: i-aws
 i-aws:
-	INTEGRATION_USE_REAL_AWS=1 ../integration.sh $(EXE_IT) -s $(NAME).integration-aws.yaml -i ../integration.yaml $(WIRE_INTEGRATION_TEST_OPTIONS)
+	INTEGRATION_USE_REAL_AWS=1 ../integration.sh $(EXE_IT) -s $(NAME).integration-aws.yaml -i ../integration.yaml -b ../brig/brig.integration.yaml  $(WIRE_INTEGRATION_TEST_OPTIONS)
 
 .PHONY: i-list
 i-list:
-	$(EXE_IT) -s $(NAME).integration.yaml -i ../integration.yaml -l
+	$(EXE_IT) -s $(NAME).integration.yaml -i ../integration.yaml -b ../brig/brig.integration.yaml  -l
 
 i-%:
-	../integration.sh $(EXE_IT) -s $(NAME).integration.yaml -i ../integration.yaml --match="$*" $(WIRE_INTEGRATION_TEST_OPTIONS)
+	../integration.sh $(EXE_IT) -s $(NAME).integration.yaml -i ../integration.yaml -b ../brig/brig.integration.yaml  --match="$*" $(WIRE_INTEGRATION_TEST_OPTIONS)
```

Comment:
This changeset doesn't look like it belongs into this PR; please extract into separate PR.

Reply:
Removed it. Parsing the brig options also introduced the dependency on brig.